data-center-and-power-discussion.md

```diff
@@ -1,5 +1,45 @@
-# Data Center
+# Data Centers
 
-# Power
+## Telemetry
 
-# Energy Efficiency
+### Open Telemetry
+
+* https://opentelemetry.io/
```

Comment:
let's have an appendix of projects and cross reference them here

Reply:
+1